Example/PeakCoreDataExample/PeakCoreDataExample/EventDetailViewController.swift

```diff
@@ -46,4 +50,20 @@ class EventDetailViewController: UIViewController, PersistentContainerSettable {
         viewContext.delete(event)
         saveViewContext()
     }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return event.attendees?.count ?? 0
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: AttendeeTableViewCell.cellIdentifier, for: indexPath) as! AttendeeTableViewCell
+        let attendee = event.attendees?.allObjects[indexPath.row] as! Person
```

Comment:
I know this isn't particularly relevant, but as `attendees` is a Set this will display the attendees in a random order each time. Better to fetch the attendees that are related to the passed in event with a sort descriptor.

Reply:
Changed the display of attendees to use data source in 97fec7b.